tests/expected/dead-invalid-access-via-raw/main.expected

```diff
@@ -1,7 +1,5 @@
 SUCCESS\
 address must be a multiple of its type's alignment
-FAILURE\
```

Comment:
Does a dereference failure now block further checks in the execution path?

Reply:
Correct - all other checks will be marked "UNKNOWN".